promgen/__init__.py

```diff
@@ -16,27 +16,3 @@
 PROMGEN_CONFIG_FILE = pathlib.Path(
     os.environ.get("PROMGEN_CONFIG", str(PROMGEN_CONFIG_DIR / "promgen.yml"))
 )
-
-# In the future we want to remove the dependency on envdir, but this will
-# require some cleanup and documentation changes, so for now we will just
-# ensure that it can't fail here if envdir is not already installed
-if PROMGEN_CONFIG_DIR.exists():
-    try:
-        import envdir
-    except ImportError:
-        pass
-    else:
-        envdir.open(PROMGEN_CONFIG_DIR)
-
-
-if "SECRET_KEY" not in os.environ:
-    try:
-        from django.utils.crypto import get_random_string
-        import warnings
-
-        warnings.warn("Unset SECRET_KEY setting to random for now")
-        os.environ["SECRET_KEY"] = get_random_string(
-            50, "abcdefghijklmnopqrstuvwxyz0123456789!@#$%^&*(-_=+)"
-        )
-    except ImportError:
-        pass
```

Comment:
After removing this the CI is failing with:

```
1.378 django.core.exceptions.ImproperlyConfigured: Set the SECRET_KEY environment variable
```

Perhaps we should set a random `SECRET_KEY` env variable in the GitHub Actions workflow file.

Reply:
I'm setting it in the docker build step for now.